<launch>
  <node cwd="node" type="joint_chain1_node" pkg="ode1" name="chain_sim">
    <param name="texture_path" value="$(find ode1)/config/textures" />
<!--     <param name="winx" value="854" /> -->
<!--     <param name="winy" value="480" /> -->
    <param name="winx" value="400" />
    <param name="winy" value="400" />
  </node>
</launch>
